app/assets/stylesheets/pages/login.scss

```diff
@@ -0,0 +1,32 @@
+///login page styleing
+
+#login {
```

Comment:
as a general css rule, don't define id selectors in CSS and use class names. It's honestly not that bad, but you get a bad look in the industry if you use css like this because 

1. most people are of object-oriented background, and like the concept of usage of classes with hierarchy.
2. The performance difference id and class is VERY small
3. Specificity of id selector can screw you up on overriding certain selectors later down the road due to css priority

Reply:
Are you saying don't use id selectors entirely or are you saying just don't create ids if they share a class. Furthermore would It make more since to just make the id a class or maybe change the name to something more specific. I used an id because I was thinking there will only be one login. 
- Is it the name that should change?

- should it be changed to a class?

Reply:
nvm I read your thing on slack I get it no classes.